src/annotator/util/clean-context.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Export clean versions of standard functions and objects.
+ *
+ * This module exports "clean" versions of various standard functions that may
+ * have been modified in the parent browsing context that the client is
+ * injected into. Code should always use the clean versions from this module,
+ * and never the potentially modified versions from the parent context.
+ *
+ * @module util/clean-context
+ */
+
+'use strict';
+
+// Append an invisible <iframe> to the parent document in order to get a clean
+// browsing context to extract clean objects and functions from.
+var iframe = document.createElement('iframe');
+iframe.style.display = 'none';
+document.body.appendChild(iframe);
```

Comment:
Done - I've changed it to not inject an iframe if it doesn't think bind has been modified